CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # jsonld-signatures ChangeLog
 
+## 4.2.0 -
+### Changed
```

Comment:
Needs empty line above `#Changed` and it should say `#Removed`. This is also a breaking change, so it will have to be version 5, unfortunately.

Reply:
no problem